src/ripple/beast/clock/tests/beast_abstract_clock_test.cpp

```diff
@@ -32,59 +32,55 @@ class abstract_clock_test : public unit_test::suite
 {
 public:
     template <class Clock>
-    void test (abstract_clock<Clock>& c)
+    void test (std::string name, abstract_clock<Clock>& c)
```

Comment:
The interface is still wrong (for nitpicky versions of wrong). `const &` is just as efficient for rvalue parameters (in this case) and saves a copy for lvalues. _If_ this function also moved from `name` into another variable then I like the interface.